tests/test_20_use_case_topology.py

```diff
@@ -985,3 +985,68 @@ def test_120_l2vpn_provisioning_failure(self):
         response = requests.get(API_URL)
         assert response.status_code == 200, response.text
         assert l2vpn_id not in response.json()
+
+    #@pytest.mark.xfail(reason="Code 201 returned after modifying the VLAN range to [1-6000]")
+    def test_141_changing_invalid_vlan_range(self):
+        """
+        Change VLAN range to an invalide range.
+        """
+
+        interfaces_id = 'aa:00:00:00:00:00:00:01:50'
+        interface_name = 'Ampath1-eth50'
+        ampath_api = KYTOS_API % 'ampath'
+        api_url_ampath = f'{ampath_api}/topology/v3'
+          
+        sdx_api = KYTOS_SDX_API % 'ampath'
+        response = requests.get(f"{sdx_api}/topology/2.0.0")
+        assert response.status_code == 200
+        data = response.json()
+        for node in data['nodes']: 
+            for port in node['ports']:
+                if port['name'] == interface_name:
+                    first_vlan_range = port['services']['l2vpn-ptp']['vlan_range']
+
+        payload = {
+            "sdx_vlan_range": [[1,6000]]
+        }
+        response = requests.post(f"{api_url_ampath}/interfaces/{interfaces_id}/metadata", json=payload)
+        ### ERROR: code 201 received with an invalid vlan range.
+        #assert response.status_code != 201, response.text
+
+        time.sleep(5)
+          
```

Comment:
we should check the topology to make sure it was not updated with the wrong VLAN range

Reply:
Please, could you explain it further? I might not have understood correctly. Notice that the topology is evaluated in this test, but that part of the code was commented out because it fails when the VLAN range is incorrectly updated to an invalid value. I have now uncommented it and added an xfail annotation to the test.

Reply:
I see. Well in this case I believe it is just a matter of uncommenting that code. Wdyt?